src/drive/actions/async.js

```diff
@@ -7,10 +7,11 @@ class Stack {
     const folder = await cozy.client.files.statById(folderId, false, {
       limit: FILES_FETCH_LIMIT
     })
+
     const parentId = folder.attributes.dir_id
     const parent =
       !!parentId &&
-      (await cozy.client.files.statById(parentId).catch(ex => {
+      (await cozy.client.files.statById(parentId, false).catch(ex => {
```

Comment:
Forces `statById ` to use the stack, especially useful while the first replication is underway.